src/utils.h

```diff
@@ -215,4 +216,18 @@ extern TSDLLEXPORT bool ts_data_node_is_available(const char *node_name);
 
 extern TSDLLEXPORT AttrNumber ts_map_attno(Oid src_rel, Oid dst_rel, AttrNumber attno);
 
+/*
+ * Return Oid for a schema-qualified relation.
+ */
+static inline Oid
+ts_relation_oid(char *schema_name, char *relation_name, bool schema_missing_ok)
```

Comment:
Since you can't overload functions, you might want to call it something like `ts_relname_reloid`. Otherwise you would have to rename this if you want to find the reloid from, say, a `Relation` in the future.

Reply:
What about `ts_get_relation_reloid` ???

Reply:
> What about `ts_get_relation_reloid` ???

Well... feel free to push the current patch, no changes necessary, but I was mostly thinking about if you want to have similar functions that accept other parameters representing relations, such as `Relation`, `RangeVar`, etc. all returning an `Oid`.